### ISSUE TITLE
Invalid read in r_list_purge()

### ISSUE THREAD

**Issue by fumfel** [radareorg/radare2#15135]
| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Ubuntu 18.04.3 x64 + Clang 9
| File format of the file you reverse (mandatory)      | PE
| Architecture/bits of the file (mandatory)            | N/A
| r2 -v full output, **not truncated** (mandatory)     | radare2 4.0.0-git 22872 @ linux-x86-64 git.3.9.0-58-gdfb4f2b06 commit: dfb4f2b06e00c8055a328cc6410a85be4c5cc0aa build: 2019-09-26__13:02:11

### Expected behavior

No crash in ASAN compiled project.

### Actual behavior

Invalid read in ASAN and SIGSEGV.

### Steps to reproduce the behavior 
- Download binary from https://github.com/radareorg/radare2-regressions/pull/1959
- Run radare2 (compiled with ASAN): `r2 -A -qq r2_ir_r_list_purge`

### Additional Logs, screenshots, source-code,  configuration dump, ...

ASAN report:

```
==7061==ERROR: AddressSanitizer: SEGV on unknown address 0x00002200003d (pc 0x7f0771b7e0e3 bp 0x7ffe43de6530 sp 0x7ffe43de6510 T0)
==7061==The signal is caused by a READ memory access.
    #0 0x7f0771b7e0e2 in r_list_purge radare2/libr/util/list.c:71:21
    #1 0x7f0771b7e315 in r_list_free radare2/libr/util/list.c:81:3
    #2 0x7f076fa17934 in object_delete_items radare2/libr/bin/bobj.c:39:2
    #3 0x7f076fa1786d in r_bin_object_free radare2/libr/bin/bobj.c:62:3
    #4 0x7f076fa104d7 in r_bin_file_free radare2/libr/bin/bfile.c:688:2
    #5 0x7f0771b7e280 in r_list_delete radare2/libr/util/list.c:105:3
    #6 0x7f0771b7e0f7 in r_list_purge radare2/libr/util/list.c:72:3
    #7 0x7f0771b7e315 in r_list_free radare2/libr/util/list.c:81:3
    #8 0x7f076f9f8384 in r_bin_free radare2/libr/bin/bin.c:520:3
    #9 0x7f07703802e1 in r_core_fini radare2/libr/core/core.c:2900:12
    #10 0x7f076d13d176 in r_main_radare2 radare2/libr/main/radare2.c:1563:2
    #11 0x7f076c91eb96 in __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310
    #12 0x55deed3da589 in _start (r2asan/binr/radare2/radare2+0x1c589)

AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: SEGV radare2/libr/util/list.c:71:21 in r_list_purge
==7061==ABORTING
```

**Comment by Guru-98** [comment]
This is solved by 42a233ab7508c14e50f5351741e2684376addfd1 from  #15175. Please close

**Comment by radare** [comment]
fixed already a while ago
